docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1073 @@
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+
+<a name="commands"></a>
+## Basic managing commands
+
+To create a command group and commands:
+
+1.  Create a command group with a command group handle.
+    The command group can include many commands, which each have a command text and type.
+    The group can have both the `Background`, `Exclusive`, `Foreground`, `System`, `SystemBackground`, and `Widget` type commands (the types are defined in the [Tizen.Uix.VoiceControlManager.CommandType](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.CommandType.html) enumeration.
+
+    > **Note**   
+	It has command priority in order of `system`, `widget`, `foreground`, `system_background` and `background`.   
+    Normally, The `system` and `system_background` commands is only registered in voice control manager.   
+    The `foreground` and `background` commands are used in the vc client app.   
+    The `exclusive` coomands is used starting with `VoiceControlManagerClient.Start(true)`.   
+    The `widget` command is automatically registered in the elementary on the screen.   
+
+    You can access the command group after you set it to the voice control and when you get the recognition result.
+
+    ```
+    void CreateCommandGroup()
+    {
+        try
+        {
+            VoiceCommandsGroup group = new VoiceCommandsGroup();
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+2.  Create a command.
+
+    First create a command handle, and then define the command and type.
+
+    ```
+    void CreateCommand()
+    {
+        try
+        {
+            VoiceCommand command = new VoiceCommand();
+
+            command.Command = "open";
+            command.Type = CommandType.Foreground;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+3.  Add the command to the command group.
+
+    If necessary, you can also remove commands from the command group.
+
+    ```
+    void AddCommand()
+    {
+        try
+        {
+            VoiceCommandsGroup group = new VoiceCommandsGroup();
+            VoiceCommand command = new VoiceCommand();
+
+            command.Command = "open";
+            command.Type = CommandType.Foreground;
+
+            group.Commands.Add(command);
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+
+    void RemoveCommand()
+    {
+        try
+        {
+            VoiceCommand command = new VoiceCommand();
+
+            command.Command = "open";
+            command.Type = CommandType.Foreground;
+
+            /// Other actions
+
+            group.Commands.Remove(command);
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+    To retrieve the commands you have added to the command list:
```

Comment:
Suggested rephrase:

To retrieve the commands, you have to add the command list: